deploy/charts/trust-manager/templates/certificate.yaml

```
@@ -24,6 +24,10 @@ spec:
   dnsNames:
   - "{{ include "trust-manager.name" . }}.{{ include "trust-manager.namespace" . }}.svc"
   secretName: {{ include "trust-manager.name" . }}-tls
+  {{- with .Values.app.webhook.tls.secretTemplate }}
```

```suggestion
  {{- with .Values.app.webhook.tls.certificate.secretTemplate }}
```

Thinking a bit more about this, I think we should make it clear that this only applies if using cert-manager to provision webhook TLS certificate. We already have a Helm alternative, and I think we might add other options later on.

---

Yep, I agree with you on this. I will update to be more explicit.